### PR TITLE
[TTAHUB] Frontend for TR dashboard

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change
Add three graphs to the TR regional dashboard (/regional-dashboard/training-reports)

## How to test
- Confirm that the graphs render, show data (derived from completed sessions), and the functionality on each works, like the table views and the screenshot buttons.
- Confirm that it mostly matches the mockup

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2752
* https://jira.acf.gov/browse/TTAHUB-2756
* https://jira.acf.gov/browse/TTAHUB-2758

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/components/WidgetH2.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function WidgetH2({ children, classNames }) {
```
Comment (AdamAdHocTeam) [inline frontend/src/components/WidgetH2.js:4]: Seems weird to have a wrapper around h2, but ok.

File: frontend/src/widgets/ReasonList.js [hunk 2]
```diff
@@ -19,20 +19,22 @@ const renderReasonList = (data) => {
...
       loading={loading}
       loadingLabel="Reason list loading"
-      title="Reasons in Activity Reports"
+      title={title}
       renderData={renderReasonList}
     />
   );
 }
 
-ReasonList.propTypes = {
```
Comment (AdamAdHocTeam) [inline frontend/src/widgets/ReasonList.js:35]: should we rename file to match widget name?

**Review by AdamAdHocTeam — COMMENTED** [review]
Not sure how feasible it is but figma shows the graph lines carry over to this white space.

Maybe filling the graph in this space would clean it up.

image

**Review by AdamAdHocTeam — COMMENTED** [review]
Again not sure if we can align these bottoms on squish of the window.

image

**Review by AdamAdHocTeam — COMMENTED** [review]
Nitpick on squishing window:

image

**Review by AdamAdHocTeam — COMMENTED** [review]
Overall this is a really good first pass. I left some comments of UI things that I think will come up on design review. I also spot checked a few of the numbers (again) in the widgets, they look good.

Also ci security scan seems to be failing but I think you looking into that already.

**Comment by thewatermethod** [comment]
> ci security scan seems to be failing but I think you looking into that already.

This is happening on all builds, FYI

I addressed your other issues with the graph resizing, @AdamAdHocTeam, so this is ready for another look

**Review by AdamAdHocTeam — APPROVED** [review]
Nice all issues fixed.

**Pull request merged**